### ISSUE TITLE
force_sort_within_sections with multi-line file docstring causes comments to be duplicated?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [timothycrosley/isort#1328]
In https://github.com/RobotLocomotion/drake/pull/13709#pullrequestreview-451092149, I noticed that a comment was getting duplicated.

Managed to get a min repro here, using `isort==5.1.2`:
https://github.com/EricCousineau-TRI/repro/blob/e73591778217dff2937db6eae5155e5574d184c0/bug/isort_bug/code.py

Basically, the following source code:
```py
"""
Multi-line docstring
"""
# Comment for A.
import a
# Comment for B - not A!
import b
```
gets transformed into this:
```py
"""
Multi-line docstring
"""
# Comment for B - not A!
import a
# Comment for B - not A!
import b
```

(see the linked `code.py` for comparison)

**Comment by timothycrosley** [comment]
Thanks for reporting! This is fixed in the 5.1.3 patch release

**Comment by EricCousineau-TRI** [comment]
Sweet! Thanks for the speedy response!

(Just an xref: bf47b311f854d25a9161e9ae6cda709002bf3f77)

**Comment by EricCousineau-TRI** [comment]
BTW I'm guessin' ya may already know, but here're GitHub's "magic words" for auto-closing issues via PRs or commits:
https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

I think this would've been auto-closed if you removed the intermediate "issue", e.g. "Fixed #1328: ..."
